lib/components/Tabs/style.scss

```diff
@@ -1,24 +1,61 @@
 @import '../../theme/tokens/facade';
 
 .nav-item {
+	$sel: &;
+
 	display: block;
 	outline: none;
 	width: auto;
 	border: none;
-	margin-bottom: -1px;
+	margin: 0 0 -1px;
 	background: transparent;
 	font-size: var(--global--fontSize--4);
-	padding: var(--global--space--3) var(--global--space--4);
-	color: getColour(gray, 400);
+	font-weight: 700;
+	line-height: 18px;
+	padding: calc(var(--global--space--3) + 1px) 0;
+	color: getColour(gray, 500);
 	cursor: pointer;
 	border-bottom: 2px solid transparent;
```

Comment:
Maybe we can use a css var to keep this border width and negative margining consistent